2.6/test/run

```diff
@@ -14,6 +14,7 @@ IMAGE_NAME=${IMAGE_NAME-centos/mongodb-26-centos7-candidate}
 CIDFILE_DIR=$(mktemp --suffix=mongodb_test_cidfiles -d)
 
 function cleanup() {
+    set +e
```

Comment:
why you need this for cleanup? This means that when we fail remove one cidfile, we exit and keep the other cidfiles around